launcher/image/preload.sh

```diff
@@ -128,6 +128,12 @@ main() {
          "Only 'debug' and 'hardened' are supported."
     exit 1
   fi
+
+  # make sure files are copied
+  sync
```

Comment:
Why is syncing required? Should we only sync the copy directory?

Reply:
So the symptom we saw in some image builds are files with 0 length, I thought this might be because of file didn't copy through and the partition got umounted, but I'm not 100% sure.

Reply:
I changed to sync the oem directory only